api/client/system/info.go

```diff
@@ -214,6 +214,7 @@ func runInfo(dockerCli *client.DockerCli) error {
 	}
 
 	fmt.Fprintf(dockerCli.Out(), "Live Restore Enabled: %v\n", info.LiveRestoreEnabled)
+	fmt.Fprintf(dockerCli.Out(), "Default Isolation: %v\n", info.Isolation)
```

Comment:
More meant; don't show this line on Linux (or when empty), to keep the output clean

> On 30 Jul 2016, at 18:53, John Howard <notifications@github.com> wrote:
> 
> In api/client/system/info.go:
> 
> > @@ -214,6 +214,7 @@ func runInfo(dockerCli *client.DockerCli) error {
> >  	}
> >  
> >  	fmt.Fprintf(dockerCli.Out(), "Live Restore Enabled: %v\n", info.LiveRestoreEnabled)
> > +	fmt.Fprintf(dockerCli.Out(), "Default Isolation: %v\n", info.Isolation)
> Technically Linux supports multiple isolation modes too, just the only one used is default. I think splitting into platform specific here is something for a follow on PR as there's things the other way around which could be tidied up.
> 
> As for where in the output, no preference. I can put it anywhere you like 😄
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>